### DISCUSSION TITLE
Weird glyph on the intersection of tree - toggleterm and buffer

### DISCUSSION THREAD

**Discussion by Raagh** [nvim-tree/nvim-tree.lua#1790] [Q&A]
Hi so do you know what this could be? it happens if I have nvim-tree open and toggleterm open. Both separately it it fine. Seems to be on the tree side. If you see right where the tree, the terminal and the buffer connect there is a |- of another color and background. Very strange

![imagen](https://user-images.githubusercontent.com/8405459/204805086-72405cd6-49a7-4a16-baa0-69f8af0e44fd.png)

**Comment by alex-courtis** [comment]
It looks like we might have a strange collision with `WinSeparator`. Do you have any customisation of those?

Also note that nvim recently deprecated `VertSplit` in favour of `WinSeparator` and some plugins might not have updated.
